packages/x-date-pickers-pro/src/DateRangeCalendar/DateRangeCalendar.test.tsx

```diff
@@ -24,7 +24,7 @@ import { describeConformance } from 'test/utils/describeConformance';
 import { RangePosition } from '../models';
 
 const getPickerDay = (name: string, picker = 'January 2018') =>
-  getByRole(screen.getByRole('grid', { name: picker }), 'gridcell', { name });
+  rtlGetByRole(screen.getByRole('grid', { name: picker }), 'gridcell', { name });
```

Comment:
`rtl` confused me at first, because I thought "Why would we need a different `getByRole` for Right-to-Left"? 😅
But since rtl stands for React Testing Library, isn't the naming still misleading given that we don't import it from the RTL package?

On a different note, why avoiding ESLint errors here?

Reply:
> On a different note, why avoiding ESLint errors here?

Would you suggest adding an ESLint ignore comment?
In this case, we want to use a root level getter and provide a specific container for it.
In other words, we are knowingly using the root getter instead of the one provided by `screen`.

Reply:
> `rtl` confused me at first, because I thought "Why would we need a different `getByRole` for Right-to-Left"? 😅
> But since rtl stands for React Testing Library, isn't the naming still misleading given that we don't import it from the RTL package?

Agreed, I didn't think of `rtl` like that initially. 🙈 
Will alias it as `rootGetByRole` instead. 👍

Reply:
Perhaps use `within` instead?

```diff
- rtlGetByRole(screen.getByRole('grid', { name: picker }), 'gridcell', { name });
+ within(screen.getByRole('grid', { name: picker })).getByRole('gridcell', { name });
```

Reply:
@cherniavskii Thanks for the reminder, I completely forgot about `within`. 👍 🙈 🤷